tdrs-backend/tdpservice/settings/common.py

```diff
@@ -179,6 +179,8 @@ class Common(Configuration):
     ]
 
     # Logging
+    # set level as 'INFO' if env var is not set
+    LOGGING_LEVEL = os.getenv('LOGGING_LEVEL', 'INFO')
```

Comment:
I would also suggest defining this in `.env.example` so we have a full list of available variables.